src/lekcja10_zadanie2_kalkulator/ShapeCalculator.java

```diff
@@ -0,0 +1,33 @@
+package lekcja10_zadanie2_kalkulator;
+
+public class ShapeCalculator extends LineCalc implements Calc2D, Calc3D {
+
+    @Override
+    public double lineLength(Line2D line) {
+
+        return super.lineLength(line);
+    }
```

Comment:
to jest niepotrzebne. metoda zostanie odziedziczona

Reply:
Poprawione, cześć dodatkowa zrobię dziś w nocy lub jutro rano.